test/crypto.js

```diff
@@ -72,20 +72,53 @@ describe('crypto', () => {
       assert.equal(input, decoded)
     })
   })
-  describe('sign and verify', () => {
-    it('should work together', async () => {
-      let privateKey = Buffer.from(privateKeyAsHex, 'hex')
-      let signature = Crypto.sign(validUnsignedTx, privateKey)
-      let pub = Buffer.from(Crypto.decodeBase58Check(publicKey.split('$')[1]))
-      let verified = Crypto.verify(validUnsignedTx, signature, pub)
-      assert.isTrue(verified)
+
+  describe('sign', () => {
+    it('should produce correct signature', () => {
+      let s = Crypto.sign(txBinary, privateKey)
+      expect(s).to.eql(signature)
     })
   })
+
   describe('verify', () => {
-    it('should verify tx with correct signature', async () => {
-      let pub = Buffer.from(Crypto.decodeBase58Check(publicKey.split('$')[1]))
-      let result = Crypto.verify(Buffer.from(txBinary), Buffer.from(signature), pub)
+    it('should verify tx with correct signature', () => {
+      let result = Crypto.verify(txBinary, signature, publicKey)
       assert.isTrue(result)
     })
   })
+
+  describe('personal messages', () => {
+    const message = 'test'
+    const messageSignatureAsHex = '0d969f938571d1436106af659bc1893e7efeda685d7f096be39d0f2c68f712e4e9ba7551df31350a4a6d07943fbfc1cbe0acfe8271cd2dcdaac3274168e53901'
+    const messageSignature = Buffer.from(messageSignatureAsHex, 'hex')
+
+    const messageNonASCII = 'tæst'
+    const messageNonASCIISignatureAsHex = '0aad557b424b33f4a2e415339d7ee143dea1f00c4a7c6c8680e1f70ff21e42b17e0e3729420daaed962999e089a18d099d39dc4a5ab4598d2b062106b59f1d01'
+    const messageNonASCIISignature = Buffer.from(messageNonASCIISignatureAsHex, 'hex')
+
+    describe('sign', () => {
+      it('should produce correct signature of message', () => {
+        const s = Crypto.signPersonalMessage(message, privateKey)
+        expect(s).to.eql(messageSignature)
+      })
+
+      it('should produce correct signature of message with non-ASCII chars', () => {
+        const s = Crypto.signPersonalMessage(messageNonASCII, privateKey)
+        expect(s).to.eql(messageNonASCIISignature)
+      })
+    })
+
+    describe('verify', () => {
+      it('should verify message', () => {
+        const result = Crypto.verifyPersonalMessage(message, messageSignature, publicKey)
+        assert.isTrue(result)
+      })
+
+      it('should verify message with non-ASCII chars', () => {
+        const result = Crypto.verifyPersonalMessage(
+          messageNonASCII, messageNonASCIISignature, publicKey)
```

Comment:
Please remove newline

Reply:
Don't you have any limits on line length? Usually, I'm trying not to make lines longer than 100 chars.